src/MudBlazor/TScripts/mudInput.js

```diff
@@ -12,3 +12,7 @@ class MudInput {
 }
 
 window.mudInput = new MudInput();
+
+window.focusInput = (elementId) => {
+    document.getElementById(elementId).focus();
```

Comment:
After focusing, you need to manually trigger the click event.
To prevent a click loop, ensure that the currently focused element is **not** an input element before triggering the click.

```suggestion
    const input = document.getElementById(elementId);
    if (input && document.activeElement !== input) {
        input.focus();
        input.click();
    } 
```